src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -401,6 +407,7 @@ struct SvgLoaderData
     Array<SvgNode *> stack = {nullptr, 0, 0};
     SvgNode* doc = nullptr;
     SvgNode* def = nullptr;
+    SvgNode* cssStyle = nullptr;
```

Comment:
if there will be more than one style element in the file, I'd like to use this var to know which one was the last one - like for gradients.